ingestion_server/ingestion_server/utils/sensitive_terms.py

```diff
@@ -0,0 +1,15 @@
+from http.client import HTTPResponse
+from urllib.request import urlopen
+
+from decouple import config
+
+
+SENSITIVE_TERMS_URL = config(
+    "SENSITIVE_TERMS_URL",
+    default="http://localhost:8001/static/mock_sensitive_terms.txt",
+)
+
+
+def get_sensitive_terms() -> list[str]:
+    response: HTTPResponse = urlopen(SENSITIVE_TERMS_URL)
+    return [line.decode("utf-8") for line in response.readlines()]
```

Comment:
I'm a little surprised this service didn't have `requests` available! Glad it's still easy enough as-is.